tests/test_string.py

```diff
@@ -38,7 +43,15 @@ def test_set(self):
         self.assertTrue(self.r.set(self.k1, self.v1))
         v1 = self.r.get(self.k1)
         self.assertEqual(self.v1, v1, '{} != {}'.format(v1, self.v1))
-
+    
+    def test_setbit(self):
+        ret = self.r.setbit(self.bitKey, self.bitPos1, self.bitVal1)
+        self.assertTrue(ret==0 or ret==1)
```

Comment:
Can you use `self.assertEqual` function for assert the result.
And in each unit test, the result will be a `certain` one instead of `ret==0 or ret==1`.